Assignments/Program.cs

```diff
@@ -1 +1,60 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T7 - Chess Board Printing
+// ---------------------------------------------------------------------------------------
+using System;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T7 {
+   public static void Main () {
+      Console.OutputEncoding = System.Text.Encoding.UTF8;
+      Console.WriteLine ("\n\x1B[4m" + "Printing 8x8 Chess Board:\n" + "\x1B[0m");
+      Console.Write ("In which way do you want the Chess Board be printed.\n\t1. WITH Unicode\n\t2. WITHOUT Unicode\nEnter (1) or (2): ");

```

Comment:
You are saying when user press 2, you are not using Unicode. But you are using pieces which has unicode characters.

Reply:
Mistake:  "Unicode" --> "Box Unicode".  I will correct in the next commit.